internal/interpolate/generated.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -e
+
+echo -e "// Code generated by ragel\n// @generated" | cat - parse.go > tmp
```

Comment:
I think, in addition to mentioning ragel, you should say that the code was generated yarpc.